sources/Outputs/include/Job.h

```diff
@@ -72,28 +67,19 @@ class Job {
                   const std::string& baseFilename) :
         filename(filename),
         dynawoLogLevel(lvl),
+        startTime(config.getStartTime()),
+        stopTime(config.getStopTime()),
         contingencyId(contingencyId),
-        baseFilename(baseFilename) {
-      initFromConfig(config);
-    }
+        baseFilename(baseFilename),
+        precision_(config.getPrecision()) {}
 
     std::string filename;                        ///< filename of the job output file
     std::string dynawoLogLevel;                  ///< Dynawo log level, in string representation
-    Seconds startTime = Seconds(0);              ///< The start time of the simulation
```

Comment:
Why removing the default values for those parameters?

Reply:
Because they are already defined for the Configuration. Since all the constructors have a Config as parameter, we would be defining default values in two different places